omod-1.9/src/test/java/org/openmrs/module/webservices/rest/web/v1_0/controller/openmrs1_9/ConceptController1_8Test.java

```diff
@@ -313,6 +317,32 @@ public void shouldSearchAndReturnConceptsThatContainsNamePartInRequest() throws
 		
 		assertThat(hits, contains(hasUuid("15f83cd6-64e9-4e06-a5f9-364d3b14a43d")));
 	}
+
+	@Test
+	public void shouldSearchAndReturnAllConceptsFromConceptClassUuid() throws Exception {
+		String conceptClassUuid = "3d065ed4-b0b9-4710-9a17-6d8c4fd259b7"; // DRUG
```

Comment:
Refer to my comment [above](https://github.com/openmrs/openmrs-module-webservices.rest/pull/550/files#r866404654)